packages/datx-jsonapi-angular/package.json

```diff
@@ -1,8 +1,32 @@
 {
   "name": "@datx/jsonapi-angular",
-  "version": "2.6.1",
+  "version": "3.0.0-beta.0",
+  "description": "DatX mixin for Angular JSON API support",
+  "main": "dist/bundles/datx-jsonapi-angular.umd.js",
```

Comment:
This should not be here. This is what the build process generates and should be present only in the generated dist/package.json file. Not sure how it got commited to this package.json file.

Reply:
I probably added it manually during conflict resolution. Will remove it again